internal/display/rsl.go

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package display
+
+import (
+	"fmt"
+
+	"github.com/gittuf/gittuf/internal/rsl"
+	"github.com/go-git/go-git/v5/plumbing"
+)
+
+// ShowRSL logs out the entries and annotations into a specified file
+func ShowRSL(entries []*rsl.ReferenceEntry, annotationMap map[plumbing.Hash][]*rsl.AnnotationEntry) string {
```

Comment:
I'd rename this function to indicate it prepares the output that ought to be displayed.

Reply:
Fixed